settings.py

```diff
@@ -1087,13 +1079,8 @@
 }
 
 ####### Couch Forms & Couch DB Kit Settings #######
-from settingshelper import (
-    get_dynamic_db_settings,
-    CouchSettingsHelper,
-    SharedDriveConfiguration
-)
-
-_dynamic_db_settings = get_dynamic_db_settings(
+COUCH_DATABASE_NAME = helper.get_db_name(COUCH_DATABASE_NAME, UNIT_TESTING)
```

Comment:
Couch test database names are now prefixed with `test_` (like SQL dbs) instead of suffixed. This changes the database name directly in the settings instead later by the test runner, so code that reads this setting prior to the test runner munging the name will not accidentally write to a production database.